### ISSUE TITLE
Recently-opened DM does not appear in room list

### ISSUE THREAD

**Issue by richvdh** [vector-im/element-web#22030]
### Steps to reproduce

Over the weekend, someone opened a DM with me, which I accepted via android.

I can't find that DM anywhere in the room list on element desktop. I *can* see it via search.

#### What did you expect?

Room should be near the top of the "people" list (when sorted by "activity")

#### What happened instead?

Can't find it anywhere, though it could be lost at the bottom or something

### Operating system

ubuntu 20.04

### Application version

Element version: 1.10.4 Olm version: 3.2.8

### How did you install the app?

debian package

### Homeserver

synapse 1.57ish

### Will you send logs?

Yes

**Comment by richvdh** [comment]
restarting the application made it appear in the expected place.

**Comment by turt2live** [comment]
@kittykat yes, that could be related. When it happens again, please submit debug logs.

**Comment by duxovni** [comment]
FWIW, restarting the app doesn't fix #22708, and I've been a member of that room for months, so I'm not totally confident these are the same issue; but I do see the "missing `m.room.create`" message in rageshakes here, so I could believe they're related.

**Comment by weeman1337** [comment]
The issue just popped up for me on develop for a non-DM room.

„clear cache and restart“ shows the room once. After the next reload it is gone again.

**Comment by florianduros** [comment]
Since there is not report for almost 4 years, i'm closing the issue
